activation/activation.go

```diff
@@ -148,7 +148,7 @@ func (b *Builder) loop() {
 				epoch := layer.GetEpoch(b.layersPerEpoch)
 				err := b.PublishActivationTx(epoch)
 				if err != nil {
-					b.log.Error("cannot create atx in epoch %v: %v", epoch, err)
+					b.log.Warning("cannot create atx in epoch %v: %v", epoch, err)
```

Comment:
IDK about this... Sometimes this **is** an error.

Reply:
"poet proof too soon" should not be an error.
Consider having a specific condition for this case with Info/Warning.

Reply:
@noamnelke can I identify **when** it is an error?

Reply:
I looked at `PublishActivationTx` and everything should be an error **except** `received PoET proof too soon`.

This was previously (badly) implemented, actually, and there's still a remnant: there's a separate warning inside the function. IMO, you should remove the warning inside the function and return a typed error that you'll catch here and print a warning instead of an error.

Reply:
Fixed